specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/Manifest.json

```diff
@@ -0,0 +1,909 @@
+{
```

Comment:
The file isn't referenced in the README, so Avocado is failing

Reply:
Done